src/node/rpc/frontend.h

```diff
@@ -323,6 +335,8 @@ namespace ccf
           return ctx->serialise_response();
         }
 
+        // By default, signed requests are verified and recorded, even on
```

Comment:
Should this comment be moved back to where it was?

Reply:
I felt it was very confusing there, above an assignment of that flag to false. This is really where the condition is.